src/server/scripts/Commands/cs_lfg.cpp

```diff
@@ -115,47 +114,36 @@ class lfg_commandscript : public CommandScript
         handler->PSendSysMessage(LANG_LFG_GROUP_INFO, groupTarget->isLFGGroup(),
             state.c_str(), sLFGMgr->GetDungeon(guid));
 
-        Group::MemberSlotList const& members = groupTarget->GetMemberSlots();
-
-        for (Group::MemberSlotList::const_iterator itr = members.begin(); itr != members.end(); ++itr)
+        for (Group::MemberSlot const& slot : groupTarget->GetMemberSlots())
         {
-            Group::MemberSlot const& slot = *itr;
-            Player* p = ObjectAccessor::FindPlayer((*itr).guid);
+            Player* p = ObjectAccessor::FindPlayer(slot.guid);
             if (p)
-                GetPlayerInfo(handler, p);
+                PrintPlayerInfo(handler, p);
             else
                 handler->PSendSysMessage("%s is offline.", slot.name.c_str());
         }
 
         return true;
     }
 
-    static bool HandleLfgOptionsCommand(ChatHandler* handler, char const* args)
+    static bool HandleLfgOptionsCommand(ChatHandler* handler, Optional<uint32> optionsArg)
     {
-        int32 options = -1;
-        if (char* str = strtok((char*)args, " "))
-        {
-            int32 tmp = atoi(str);
-            if (tmp > -1)
-                options = tmp;
-        }
-
-        if (options != -1)
+        if (optionsArg)
         {
-            sLFGMgr->SetOptions(options);
+            sLFGMgr->SetOptions(*optionsArg);
```

Comment:
what are the values for this? are they some enum?

Reply:
It's a bitmask. Same as the `DungeonFinder.OptionsMask` config value.